packer/scylla_install_image

```diff
@@ -211,6 +211,15 @@ WantedBy=multi-user.target
     if args.target_cloud == 'azure':
         with open('/etc/hosts', 'a') as f:
             f.write('\n\n169.254.169.254    metadata.azure.internal\n')
+        with open('/etc/ssh/sshd_config.d/50-cloudimg-settings.conf') as f:
+            cfg = f.read()
+        cfg = re.sub('ClientAliveInterval 120', 'ClientAliveInterval 180', cfg, flags=re.MULTILINE)
+        with open('/etc/ssh/sshd_config.d/50-cloudimg-settings.conf', 'w') as f:
```

Comment:
i.e. `99_scylla_client_alive.conf`

Reply:
Because `/etc/ssh/sshd_config.d/50-cloudimg-settings.conf` already have this configuration `ClientAliveInterval 120` , how adding a new configuration file will help us? i don't think it will override it

Reply:
btw, this change is only related to `Azure`, and this is based on their recommendations , so anyway it will not break AWS or GCP

Reply:
> now I saw your other comment ;-) If `ClientAliveInterval` is not available anymore, shouldn't the new configuration file will break the next releases as well? If I am not mistaken, the command in L#216 will be replaced only for a specific value, so if this is not available anymore we will not do anything, right?

yes exactly what I mean,

changing configuration in `.d` folder is always in alphabetically order, hence adding bigger number at the prefix of the files should be working.

Reply:
@fruch Changed, please review